app/assets/javascripts/hub.coffee

```diff
@@ -13,6 +13,16 @@ $(document).on 'turbolinks:load', ->
   $body = $('body')
 
   #-----------------------
+  # Anchor link seeking
+  #-----------------------
+  if window.location.hash
+    hash = window.location.hash.substring(1)
```

Comment:
till we have turbolinks issue fix we need to do this. The other option is to fix the content with data-turbolink='false'. Should we do that. This needs discussion with @abperiasamy  https://github.com/turbolinks/turbolinks/issues/75